app/assets/stylesheets/footer.sass

```diff
@@ -1,53 +1,60 @@
 @import oh-colors
 
 footer
-  background: $bds-background-black
-  border-top: 5px #DDDDDD solid
-  color: #EEE
-  font-weight: 200
-  margin: 0 -10px
-  min-height: 175px
+  background: $footer-background !important
+  margin: 20px 0 0 0
+  min-height: 250px
   padding-top: 10px
   width: 980px
-  a
-    color: white
-    &:hover
-      color: white
 
   .footer-left
     float: left
-    margin-right: 30px
-    width: 10%
-    ul li
-      margin: 0
-      a
-        color: white
-        padding: 8px 12px
-        &:hover
-          background: #0082C6
-          border-radius: 0
+    margin-top: 20px
+    margin-left: 30px
+    width: 15%
+    color: $secondary-light-purple !important
```

Comment:
Should we be using important rule as much.  It's my understanding that this is not a best practice.  Here's a link that talks about when it should be used.  https://css-tricks.com/when-using-important-is-the-right-choice/

Reply:
Agreed.  There's lots of `!important` scattered thru all of the code.  After touching up a few pages, I'll have a better feeling on how best to minimize `!important` for common UIs.